airbyte-webapp/src/views/Connector/ConnectorDocumentationLayout/ResourceNotAvailable/ResourceNotAvailable.tsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable css-modules/no-unused-class */
```

Comment:
This needs to be checked. 
Issue described here: https://www.loom.com/share/48321a7686e24f92bfb605bd279e4f9d